arangod/RocksDBEngine/RocksDBOptionFeature.cpp

```diff
@@ -1498,6 +1500,38 @@ of outgrowing the maximum number of file descriptors the ArangoDB process
 can open. Thus the option should only be enabled on deployments with a
 limited number of edge collections/shards/indexes.)");
 
+  options
+      ->addOption("--rocksdb.partition-files-for-zkd-index",
+                  "If enabled, the index data for different zkd "
+                  "indexes will end up in different .sst files.",
+                  new BooleanParameter(&_partitionFilesForZkdIndexCf),
+                  arangodb::options::makeFlags(
+                      arangodb::options::Flags::Uncommon,
+                      arangodb::options::Flags::Experimental,
+                      arangodb::options::Flags::DefaultNoComponents,
+                      arangodb::options::Flags::OnAgent,
+                      arangodb::options::Flags::OnDBServer,
+                      arangodb::options::Flags::OnSingle))
+      .setIntroducedIn(31200)
+      .setLongDescription(R"(Enabling this option will make RocksDB's
+  compaction write the persistent index data for different zkd
```

Comment:
```suggestion
  compaction write the zkd index data for different zkd
```

Reply:
I did this change initially, but I though it is refering to the persistent data of that index, which just happens to be called `persistent` index and in this case `zkd` index.